Author: LaMont Jones <lamont@ia.mmjgroup.com>
Description: Don't prototype read() and close() in mawk.h, #include <unistd.h> instead.

diff -urNad 03.mawk.tmp/mawk.h 03.mawk/mawk.h
--- 03.mawk.tmp/mawk.h	1996-11-08 14:54:52.000000000 +0000
+++ 03.mawk/mawk.h	2003-04-09 22:18:33.000000000 +0100
@@ -52,6 +52,7 @@
 
 #include  "nstd.h"
 #include <stdio.h>
+#include <unistd.h>
 #include "types.h"
 
 #ifdef   DEBUG
@@ -153,9 +154,6 @@
 int   PROTO( binmode, (void)) ;
 
 
-int   PROTO( close, (int) ) ;
-int   PROTO( read, (int , PTR, unsigned) ) ;
-
 void PROTO ( parse, (void) ) ;
 int  PROTO ( yylex, (void) ) ;
 int  PROTO( yyparse, (void) ) ;
